src/DNA.cpp

```diff
@@ -456,8 +493,31 @@ string DNA::to_string(DNA d) {
     return output;
 }//end to_string
 
+/**
+ * breed
+ * 
+ * Breeds two dna sequences randomly. 
+ * Has between [1, 5) crosses and 
```

Comment:
Explain what 'crosses' and 'mutations'. Chances are the person intently reading these comments will need a little refresh on what those are and specifically what they do to the DNA

Reply:
I have defined more clearly what crosses and mutations are in the their respective functions in the next commit.